.github/workflows/publish.yml

```diff
@@ -93,3 +102,24 @@ jobs:
           git config --global user.email "broadbot@broadinstitute.org"
           git commit -am "Auto update CBAS UI version to $HELM_NEW_TAG"
           git push https://broadbot:$BROADBOT_TOKEN@github.com/broadinstitute/cromwhelm.git main
+
+  sonarqube:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # Needed by sonar to get the git history for the branch the PR will be merged into.
+        with:
+          fetch-depth: 0
+
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+
+      - name: SonarQube scan
+        uses: actions/checkout@v2
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
I don't think we have these tokens added to our repo? You might have to add them.

Reply:
+1. Ask in devops champions (and mention "atlantis")